src/dlgIRC.cpp

```diff
@@ -196,7 +203,7 @@ QPair<bool, QString> dlgIRC::sendMsg(const QString& target, const QString& messa
 void dlgIRC::ircRestart(bool reloadConfigs)
 {
     QString msg = tr("Restarting IRC Client");
-    ircBrowser->append(IrcMessageFormatter::formatMessage("! %1.").arg(msg));
+    ircBrowser->append(IrcMessageFormatter::formatMessage(tr("! %1.", "The leading ! is a special status marker used programmatically - not an actual exclamination mark - leave unchanged!").arg(msg)));
```

Comment:
You're saying the Spanish shouldn't change it to ¡ ? :wink:

Reply:
¡Sí!